examples/atari_deepRL_saliency.ipynb

```diff
@@ -206,7 +206,7 @@
      "name": "stdout",
```

Comment:
Since we are editing this notebook, can we remove the "please" in the header section when talking about Colab? I think this was one of the changes that Mary Elise had wanted to make.

"To run this notebook in Colab, please use the link below:" -> "To run this notebook in Colab, use the link below:"